Marketplace/voiceHUDApp/App Resources/voiceHUD.app.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
make sure you are putting things in the right directory and using camelCase without spaces